android/engine/src/main/java/org/smartregister/fhircore/engine/ui/theme/Themes.kt

```diff
@@ -16,16 +16,59 @@
 
 package org.smartregister.fhircore.engine.ui.theme
 
+import androidx.appcompat.app.AppCompatDelegate
 import androidx.compose.material.MaterialTheme
-import androidx.compose.runtime.Composable
+import androidx.compose.runtime.*
+import androidx.lifecycle.ViewModel
+import androidx.lifecycle.viewmodel.compose.viewModel
 
 /**
  * TODO fix issue with ktfmt formatting annotated high order functions. Current workaround below:
  * lambda in this format content: (@Composable() () -> Unit) to allow spotlessApply
  *
  * To enable app theme set darkTheme = isSystemInDarkTheme
  */
+//@Composable
+//fun AppTheme(darkTheme: Boolean = false, content: (@Composable() () -> Unit)) {
+//  MaterialTheme(colors = if (darkTheme) DarkColors else LightColors, content = content)
+//}
```

Comment:
Actually the dark theme was disabled because we do not support it yet. Initially the parameter `darkTheme` defaulted to `isSystemInDarkTheme()` which applied the NIGHT_MODE. This change will revert to the previous implementation.

Reply:
Unless the new implementation fixes the app crash.